controller.js

```diff
@@ -68,4 +75,12 @@ const patchReview = (request, response, next) => {
     })
     .catch(next)
 }
-module.exports = { getCategories, getReviews, getReviewById, getComments, postComment, patchReview, getUsers }
\ No newline at end of file
+
+const deleteCommentById = (request, response, next) => {
+    const {comment_id} = request.params
+    deleteComment(comment_id).then(()=>{
+        response.status(204).send({status : 204})
```

Comment:
You don't need to return anything so you can literally just have response.status(204)